src/types/zodFlaw.ts

```diff
@@ -149,7 +149,7 @@ export const ZodFlawCommentSchema = z.object({
     creation_time: z.string().nullish(),
     private_groups: z
       .string()
-      .transform((jsonString: string) => JSON.parse(jsonString))
+      .transform((jsonString: string) => JSON.parse(jsonString.replace(/'/g, '"')))
```

Comment:
Wouldn't this create invalid JSON?

Reply:
```json
{
  "key": ["yes this would create 'invalid JSON'"]
}
```

Reply:
In the case of the groups, they seem to be store in the following manner: `"['group_name','other_group_name']"`, but with the way I've implemented this fix, it will fail if that is not the case.

Reply:
OK good, just checking for consideration. Deferring to @JakubFrejlach's approval

Reply:
It may make sense to just remove this line, testing now...

Reply:
Actually on second thought, I checked the data and not every FlawComment does have the `private_groups` this might be the problem why is it parsed incorrectly?